config/index.js

```diff
@@ -25,18 +33,33 @@ function printConfig (config) {
 *    ]
 *
 */
-module.exports = function (config, overrides) {
-  config.load = function () {
+module.exports = function (config, configDirPath, overrides) {
```

Comment:
I like the idea of using an object.

As for `load` - I thought it was a bit odd initially, but thought maybe you had future plans. I'm fine with getting rid of the load step if others are on board. I'll save that work for a separate PR to avoid creep.